### ISSUE TITLE
One-file .exe can't run on server (network path) with "--disable-console".

### ISSUE THREAD

**Issue by GelzoneCC** [Nuitka/Nuitka#2872]
A little example for #2864. I just made a little example to copy the issue. The code below is using selenium to access Google.com. And I use `nuitka --lto=no --standalone --onefile --include-package-data=selenium --disable-console --output-dir=BIOS --remove-output GoogleTest.py` to compile this as an .exe file.

## GoogleTest.py

```
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from webdriver_manager.chrome import ChromeDriverManager

from time import sleep

web_address = "https://www.google.com"

driver = webdriver.Chrome(service = Service(ChromeDriverManager().install()))

driver.get(web_address)

sleep(10)

driver.quit()
```

## Issue

I can run it well on local side (e.g. D:\WorkSpace\GoogleTest.exe) which means open a browser and access to Google.com then close the browser after 10 secs.
But when I move this .exe to a **network path** (e.g. \\\\srv\folder\python_exe\GoogleTest.exe) to run, it shows nothing, no browser opened, no error message showed.
After that, I tried to remove `--disable-console` and compiled it again. With window console showed, both local side and network path can run it well. 
I wonder how come will this happened?

## Python Package & Nuitka ##

Nuitka 2.2
selenium 4.10.0
webdriver-manager 4.0.1

2.2
Commercial: None
Python: 3.9.13 (tags/v3.9.13:6de2ca5, May 17 2022, 16:36:42) [MSC v.1929 64 bit (AMD64)]
Flavor: CPython Official
Executable: D:\PyWorkSpace\Python_Project\venv_UITool\Scripts\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Version C compiler: C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Tools\MSVC\14.16.27023\bin\Hostx64\x64\cl.exe (cl 14.1).

**Comment by kayhayen** [comment]
Can you use `--force-stdout` and `--force-stderr` to capture potential errors. I also forgot, but I believe they default to %NULL% with disabled console, and not %NONE% but if they do, not all code works with `pythonw.exe` which is the equivalent.

**Comment by GelzoneCC** [comment]
The output of `--force-stderr`, it shows: 
"\\\\srv\\folder\python_exe                
CMD.EXE was started with the above path as the current directory. UNC paths are not supported.
Defaulting to Windows directory."

But the error message is English version. I am Chinese user, so the real message is 
![run_on_svr_issue](https://github.com/Nuitka/Nuitka/assets/24203936/e374b7c1-36e7-4c72-b907-b88b2eb478fa)
Sorry for hiding the real path for confidentiality.

The output of `--force-stdout` was nothing, no message shows.

**Comment by GelzoneCC** [comment]
But as I mentioned before, aftre I remove the `--disable-console`, even though it shows the error message I just gave, the .exe still worked well.

**Comment by kayhayen** [comment]
Well, it probably requires a console for some reason, and if you don't disable it, there will already be one. If you dive deep and figure out, what their issue is with not having a console, great. But it's not an issue with Nuitka, more Windows and Selenium I guess. It cannot make software work in contexts, where it is not already working. And `pythonw.exe` seems unsupported by Selenium in network paths.

**Comment by kayhayen** [comment]
You can still try the current development version and `--windows-console=attach,` but I don't think it will make a difference if you launch it via Explorer. I know it's not practical, but if you used Selenium and its downloaded Browsers from the network paths, you probably would have the same issue of being unable to use `pythonw.exe` and not `python.exe`. If however, you succeed in that, there would be something to debug in Nuitka.

One thing that I have done in the past is to remove UNC paths during compilation, if possible. From there, I remember it is possible to iterate the drives and find a mount point. That is the same, but of course only if it exists, and if it doesn't, then you are still lost. Nuitka creates short paths for `__file__`, and applying those conversions would be great for both compile time and run time.

I remember spending some quality time on this. A general solution is not easy, though, but I assume even partial ones will be helpful.

That would mitigate the issue. Of course, the real fix is to make all Python software work with UNC paths, but we can only dream of that.

**Comment by GelzoneCC** [comment]
Yes, I agree with you that it seems it's not the Nuitka issue because the .exe only can't run on network path with `--disable-console`. Maybe it needs a console to run well on the network path. 
Anyway, the issue is not a big impact on the crawler tool, just sacrificing a little user experience.

Furthermore, I think this may not be my last crawler tool, so I will keep spending time figuring it out.
I will try `--windows-console=attach` you mentioned and find the way toward the `selenium` or `webdriver-manager` package. Maybe the answer is there.

Thank for your reply.
